etrago/cluster/disaggregation.py

```diff
@@ -744,6 +755,10 @@ def solve_partial_network(
                     )
                     delta = abs((new_columns.sum(axis=1) - clt).sum())
                     epsilon = 1e-5
+                    if delta > epsilon:
+                        import pdb
+
```

Comment:
I suppose you used these lines for debugging purposes. Can we exclude them now?

Reply:
I removed these lines